hw_2/task_2.py

```diff
@@ -0,0 +1,48 @@
+# Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем. 
+# Программа должна возвращать сумму и произведение* дробей. Для проверки своего кода используйте модуль fractions.
+import math
+from fractions import Fraction
+
+def input_str():
+    print()
+    s = input( 'Введите строки вида “a/b” - дробь с числителем и знаменателем   ')
+    return s
+def get_numerator_and_denominator(num: str):
+    num_lst = num.split('/')
+    numerator = int(num_lst[0])
+    denominator = int(num_lst[1])
+    return numerator, denominator
```

Comment:
Так будет чуть проще
```suggestion
    return map(int, num.split('/'))
```
Хотя если тогда вообще смысл использовать для этого отдельную функцию.